tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot.md

```diff
@@ -0,0 +1,265 @@
+
+1. Export the WebAssembly runtime for the parachain by running a command similar to the following
+ 
+      ```bash
+      ./target/release/parachain-template-node export-genesis-wasm --chain raw-parachain-chainspec.json para-4105-wasm
+      ```
+
+2. Export the genesis state for the parachain by running a command similar to the following
+
+      ```bash
+      ./target/release/parachain-template-node export-genesis-state --chain raw-parachain-chainspec.json para-4105-state
+      ```
+
+## Start the Collator Node
+
+You must have the ports for the collator publicly accessible and discoverable to enable parachain nodes to peer with Rococo validator nodes to produce blocks. You can specify the ports with the --port command-line option. For example, you can start the collator with a command similar to the following:
+
+```bash
+./target/release/parachain-template-node --collator \
+  --chain raw-parachain-chainspec.json \
+  --base-path /tmp/parachain/pubs-demo \
+  --port 50333 \
+  --rpc-port 8855 \
+  -- \
+  --execution wasm \
+  --chain rococo \
+  --port 50343 \
+  --rpc-port 9988
+```
+
+In this example, the first `--port` setting specifies the port for the collator node and the second `--port` specifies the embedded relay chain node port. The first `--rpc-port` setting specifies the port you can use to connect to collator. The second `--rpc-port` specifies the port for connecting to the embedded relay chain.
+
+## Register as a Parathread
+
+Before leasing a slot on a public relay chain to become a parachain, you must register as a parathread on Rococo.
+
+To register as a parathread, follow these steps:
+
+1. Navigate to the **Parachains** section
+    1. Click on the **Network** tab in the top menu
+    2. Select the **Parachains** option from the dropdown menu
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-5.webp) 
+
+2. Register a parathread
+    1. Select the **Parathreads** tab
+    2. Click on the **+ ParaThread** button
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-9.webp)
+
+3. Fill in the required fields:
+    - **parachain owner** - the account that will own the parachain
+    - **code** - the Wasm code of the parachain
+    - **initial state** - the genesis state of the parachain
+    - Click on the **+ Submit** button
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-10.webp)
+
+4. After submitting the transaction, you can navigate to the Explorer tab and check the list of recent events for successful `parathread.Registered`
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-11.webp)
+
+After registering as a parathread, you can lease a slot on the Rococo relay chain to become a parachain.
+
+## Request a Parachain Slot
```

Comment:
Here we should continue the tutorial with Coretime. Not sure if we want to add all the steps here or link to a different page